src/test/java/duck/task/TodoTaskTest.java

```diff
@@ -0,0 +1,38 @@
+package duck.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TodoTaskTest {
+    @Test
+    public void stringify_success() {
+        TodoTask task = new TodoTask("do laundry", false);
+        assertEquals("T010/do laundry", task.stringify());
+
+        task = new TodoTask("finish homework", true);
+        assertEquals("T115/finish homework", task.stringify());
+    }
+
+    @Test
+    public void parse_success() {
+        try {
+            TodoTask task = new TodoTask("do laundry", false);
+            String taskString = task.stringify();
+            Task parsedTask = TodoTask.parse(taskString);
+            assertEquals(task, parsedTask);
+        } catch (Exception e) {
+            fail();
+        }
+    }
+
+    @Test
+    public void toString_success() {
+        TodoTask task = new TodoTask("do laundry", false);
```

Comment:
I like how you have multiple test cases here. You can consider to add in negative cases as well?